custom_components/hacs/__init__.py

```diff
@@ -134,17 +134,12 @@ async def async_startup():
             hacs.disable_hacs(HacsDisabledReason.RESTORE)
             return False
 
-        if not hacs.configuration.experimental:
-            can_update = await hacs.async_can_update()
-            hacs.log.debug("Can update %s repositories", can_update)
-
         hacs.set_active_categories()
 
         async_register_websocket_commands(hass)
         await async_register_frontend(hass, hacs)
 
-        if hacs.configuration.experimental:
-            await hass.config_entries.async_forward_entry_setups(config_entry, [Platform.UPDATE])
+        await hass.config_entries.async_forward_entry_setups(config_entry, [Platform.UPDATE])
```

Comment:
Instead of recreating this list, we should do as core integrations here, and define a `PLATFORMS` const that could be reused.